src/Components/Countrylist/Countrycard.css

```diff
@@ -15,6 +13,22 @@
   overflow: hidden;
 }
 
+.card:nth-child(4n+1) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image1.png') center/cover;
+}
+
+.card:nth-child(4n+2) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image2.png') center/cover;
+}
+
+.card:nth-child(4n+3) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image3.png') center/cover;
+}
+
+.card:nth-child(4n+4) {
+  background: linear-gradient(rgba(209, 66, 121, 0.9), rgba(209, 66, 121, 0.9)), url('../../Images/image4.png') center/cover;
+}
+
 .card-title {
```

Comment:
- [OPTIONAL] The first reviewer asked that you make your design identical to that of the design template provided for this project. You didn't implement it. I would advise you to click on the video link provided in this comment of the first reviewer https://github.com/karanJ2212/weather_report/pull/5#discussion_r1172773478 and implement the changes suggested. 👍 😎